application-confluence-migrator-pro-default/src/main/java/com/xwiki/confluencepro/internal/ConfluenceMigrationJob.java

```diff
@@ -74,24 +69,26 @@ public class ConfluenceMigrationJob
      */
     public static final String JOBTYPE = "confluence.migration";
 
+    private static final String NPMIG = "npmig";
```

Comment:
I think `NPMIG` is not descriptive enough
```suggestion
    private static final String NPMIG_REQUEST_PREFIX = "npmig";
```